cypress/integration/liquiditymining.spec.js

```diff
@@ -22,105 +22,91 @@ describe('Liquidity Mining', () => {
   context('Widgets', () => {
     it('should show capital in farms', () => {
       cy.get(
-        ':nth-child(1) > .sc-jSFjdj > .SFMFb > .ctlBxL > .etUwLo > .QxpiM > .sc-cTJkRt'
-      ).should('be.visible')
-      cy.get(
-        ':nth-child(1) > .sc-jSFjdj > .SFMFb > .ctlBxL > .etUwLo > .fgTdQE > .sc-efHYUO'
-      ).should('not.be.empty')
-      cy.get(
-        ':nth-child(1) > .sc-jSFjdj > .SFMFb > .ctlBxL > .etUwLo > .fgTdQE > .sc-iBzEeX'
+        ':nth-child(1) > .sc-jSFjdj > .SFMFb'
```

Comment:
I couldn't get unique `data-cy` ids attached to elements in the liquidity mining page but other pages should have more IDs to target.

Reply:
to resolve this, add 2 seconds to line 5 like `cy.wait(5000)`. This page is slow, so if you wait a couple more seconds here then it should load properly

Reply:
Thanks for the tip! I just tried this with no luck. 

<img width="459" alt="Screen Shot 2021-09-08 at 8 01 20 PM" src="https://user-images.githubusercontent.com/3699047/132601060-e9cbc14b-4b74-49c3-9682-e42d56789b5e.png">

It turns out the page loads fine in 3 seconds but any `data-cy` attributes that I attach to elements in [Treasury](https://github.com/SetProtocol/index-ui/blob/master/src/views/Farm/components/Treasury.tsx) don't end up persisted. One observation I've made is that all the components that I'm trying to attach ids to are react-neu [Card](https://github.com/clintonbembryjr/react-neu/tree/master/src/components/Card) elements but I'm not sure why that could be throwing this off

Reply:
Confirmed Card components are also broken on the homepage. https://github.com/clintonbembryjr/react-neu/issues/5